src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -621,6 +622,47 @@ groovyscript.wiki.botania.rune_altar.input.required=that `input` IIngredients ca
 groovyscript.wiki.botania.rune_altar.mana.value=Sets the mana cost of processing the recipe
 
 
+# Tinkers
```

Comment:
could you put this in alphabetical order, instead of between Botania and Calculator.